### ISSUE TITLE
TypeScript: callback for unsubscribe should be optional

### ISSUE THREAD

**Issue by ogis-onishi** [mqttjs/MQTT.js#584]
https://github.com/mqttjs/MQTT.js/blob/v2.5.1/types/lib/client.d.ts#L140

```
  public unsubscribe (topic: string | string[], callback: PacketCallback): this
```

The callback should be optional as the same as the other functions such as publish, subscribe.

**Comment by sublimator** [comment]
#585

**Comment by sublimator** [comment]
Any other issues you came across ?

**Comment by ogis-onishi** [comment]
I just found #571. No other issues so far.

**Comment by ogis-onishi** [comment]
fixed by #585
